numba/tests/test_parallel_backend.py

```diff
@@ -10,29 +10,30 @@
 import os
 import sys
 import subprocess
-import signal
-
-from numba import config, utils
-if utils.PYVERSION >= (3, 0):
-    import queue as t_queue
-    import faulthandler
-else:
-    import Queue as t_queue
 
 import numpy as np
 
+from numba import config, utils
+
 from numba import unittest_support as unittest
-from numba import jit, vectorize, guvectorize, njit
+from numba import jit, vectorize, guvectorize
 
 from .support import temp_directory, override_config, TestCase, tag
 
 from .test_parfors import skip_unsupported as parfors_skip_unsupported
 from .test_parfors import linux_only
 
+
+if utils.PYVERSION >= (3, 0):
+    import queue as t_queue
```

Comment:
`queue` should probably come from `.six`?

Reply:
fixed in https://github.com/numba/numba/pull/4155/commits/37684eff506bdf7fbdaa00e373d06d9321d57409